libevmasm/Instruction.h

```diff
@@ -89,6 +89,7 @@ enum class Instruction: uint8_t
 	CHAINID,			///< get the config's chainid param
 	SELFBALANCE,		///< get balance of the current account
 	BASEFEE,            ///< get the block's basefee
+	BLOBHASH,           ///< get the blob versioned hash
```

Comment:
```suggestion
	BLOBHASH = 0x49,    ///< get a blob's versioned hash
```

There can be more than one blob in a block and the instruction only gets one of them. Or maybe we could be a bit more verbose: `get a versioned hash of one of the blobs associated with the block`.

Reply:
I believe it's better to state that it is associated with the transaction, right? Because the `index` in `blobhash(index)` represents the index of the blob within the transaction. Although EIP-4844 doesn't provide details on this, upon reviewing it and other associated EIPs like [EIP-2718: Typed Transaction Envelope](https://eips.ethereum.org/EIPS/eip-2718), it appears to be the case. Initially, I thought it was related to a beacon block number, but it doesn't seem to be the case

Reply:
Right, my bad. It's per transaction.